dist/profile/templates/jenkinscontroller/casc/clouds.yaml.erb

```diff
@@ -494,6 +496,83 @@ jenkins:
             systemctl start datadog-agent.service
           ) 2>&1 | tee /var/log/agent-init-datadog.log
 
+          ## Setup NVMe if exist
+          MNT_DIR="<%= agent['agentDir'] ? agent['agentDir'] : @jcasc['agents_setup'][$os]['agentDir'] %>"
```

Comment:
is it better to have this in a script in a repository and download it on startup? then you can shell check it and don't have to escape every line basically?

Reply:
Yes, that would avoid escaping, but it creates a dependency to another webserver. It's a good optim. but for later.